CMakeLists.txt

```diff
@@ -203,6 +203,15 @@ if (WITH_PIRANHA)
     set(WITH_BOOST yes)
 endif()
 
+# BENCHMARKS
+set(BUILD_BENCHMARKS yes
```

Comment:
I keep it `yes` now, and only build the Nonius benchmarks if `BUILD_BENCHMARKS_NONIUS` is `yes` (it is `no` by default).